alexagruso/powder-simulator
# Feature/UI

What to test:

- Make sure all colors work, and that pixels cannot be drawn outside of the board
- Make sure that color switching works
- Make sure you can always exit using Escape

If any crashes occur, try to replicate them and report them here as a comment.
